### ISSUE TITLE
Measures of Political Knowledge

### ISSUE THREAD

**Issue by rrodriguezbarron** [rrodriguezbarron/Resentment#6]
Should we add them?

**Comment by braedenmc1** [comment]
Yes! Gomez and Wilson (2008) argue less sophisticated people are unable to see systemic arguments about poverty and racism. This could be an exception because redistributive injustice is so central to rural consciousness (Cramer 2012, Lyons and Ultych 2019) despite evidence that resources are distributed equitably across geography.
